src/frontend/handlers.go

```diff
@@ -72,6 +72,13 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
+	// Feature: convert currency for product to all currencies
+	if os.Getenv("CONVERT_CURRENCIES") == "Active" {
```

Comment:
Let's change it to true/false and also make the check case insensitive.

Reply:
fixed!